docs/source/sphinxext/github_link.py

```diff
@@ -96,15 +111,14 @@ def _linkcode_resolve(domain, info, package, url_fmt, revision):
             # fn is expected to be the absolute path.
             fn = os.path.relpath(source_file, start=package)
             print("{}:{}".format(
-                os.path.abspath(os.path.join("..", "python", "cuml", fn)),
+                os.path.abspath(os.path.join("..", "python", "raft", fn)),
                 lineno))
         else:
             return
     else:
-        # Test if we are absolute or not (pyx are relative)
-        if (not os.path.isabs(fn)):
-            # Should be relative to docs right now
-            fn = os.path.abspath(os.path.join("..", "python", fn))
+        if fn.endswith(".pyx"):
+            sp_path = next(x for x in sys.path if re.match(".*site-packages$", x))
+            fn = fn.replace("/opt/conda/conda-bld/work/python/cuvs", sp_path)
```

Comment:
I think we need to make sure this builds for developers and not just by CI, right? Won't this break developers trying to build the docs locally?